src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -282,7 +282,7 @@ const CreateNewBillingProjectWizard = ({ onSuccess, billingAccounts, authorizeAn
               'You do not have access to any Google Billing Accounts. Please verify that a billing account has been created in the ' +
               'Google Billing Console and terra-billing@terra.bio has been added as a Billing Account User to your billing account.',
               div({ style: { marginTop: '0.5rem' } }, [
-                h(Clickable, {
+                h(Link, {
```

Comment:
@yunip20 I learned that we should still make this a Link to pick up some of the styling (hover effect).